tests/unit/s2n_certificate_test.c

```diff
@@ -19,6 +19,53 @@
 #include <s2n.h>
 #include "utils/s2n_safety.h"
 
+#include <openssl/crypto.h>
+#if defined(OPENSSL_IS_AWSLC)
+#include <openssl/mem.h>
+#endif
+
+struct host_verify_data {
+    const char *name;
+    uint8_t found_name;
+    uint8_t callback_invoked;
```

Comment:
I prefer `bool`s to capture what these should be used for. Types are mostly for humans to communicate intent.